tools/llhd-sim/llhd-sim.cpp

```diff
@@ -102,6 +102,10 @@ static cl::opt<TraceFormat> traceMode(
             "instance and signals not having the default name '(sig)?[0-9]*'"),
         clEnumValN(noTrace, "no-trace", "Don't dump a signal trace")));
 
+static cl::list<std::string>
+    sharedLibs("shared-libs", cl::desc("Libraries to link dynamically"),
+               cl::ZeroOrMore, cl::MiscFlags::CommaSeparated);
```

Comment:
FWIW this new flag is totally optional, and should only be required in the situations that didn't work previously.